libs/books/data-access/src/lib/+state/reading-list.selectors.ts

```diff
@@ -35,7 +35,7 @@ export const getAllBooks = createSelector<
   Record<string, ReadingListItem>,
   ReadingListBook[]
 >(getBooks, getReadingListEntities, (books, entities) => {
-  return books.map(b => ({ ...b, isAdded: Boolean(entities[b.id]) }));
+  return books.map(book => ({ ...book, isAdded: Boolean(entities[book.id]), finished: entities[book.id]?.finished }));
```

Comment:
You can update finishedDate as well here.

Reply:
Added finishedDate